experiments/crystalvc-670c2b5/ToDo.txt

```diff
@@ -1,89 +0,0 @@
--- What do we do if my repository is behind yours, but yours is in a conflicting (two headed) state?  Currently, we just say "behind".

```

Comment:
Can you go through the subjects and delete all sorts of unnecessary files like these?